carma-messenger-core/cpp_message/README.md

```diff
@@ -0,0 +1,3 @@
+# cpp_message
+
+TODO for USER: Add description of package and link to confluence documentation.
```

Comment:
Add small description and a link to this design document 
https://usdot-carma.atlassian.net/wiki/spaces/CRMMSG/pages/1320058912/Detailed+Design+-+Message